.jenkins/build_docker_images.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Set variable to allow parallel building in the Docker image creation
+export DOCKER_BUILDKIT=1
+
+# Get the dockerfiles for the images to be tested on Piz Daint
+dockerfiles=( $( ls docker/Dockerfile.* ) )
+
+for df in ${dockerfiles[@]}; do
+   # Grab the architecture tag from the dockerfile filename
+   arch=${df:18}
+
+   # Set the names of the new Docker image and tar file
+   container=us.gcr.io/vcm-ml/fv3gfs-compiled:$arch
+   tar_file=fv3gfs-compiled_${arch}.tar
+   
+   # Build the docker image and push image to VCM's Google Container Repository
+   docker build -f $df --target fv3-bld -t $container .
+   docker push $container
```

Comment:
In the future, we could build (and test) these on the cloud, and pull them for sarus testing.

Reply:
Sure, all the newly built images should run fine on the cloud.  We will still need to perform the awkward docker save/tarfile transfer process as Sarus cannot perform a pull directly from GCR.

Reply:
Yes, probably when we do that we should publish tars as well.